backend/fleet_management/admin.py

```diff
@@ -18,22 +18,27 @@ def dehydrate_passengers(self, drive):
             for passanger in drive.passengers.all()
         )
 
+    fuel_consumption = Field(attribute='fuel_consumption')
```

Comment:
Do we need to make it explicit here?

Reply:
Yes, by default, only fields from the database are imported. We want to additionally import @property of this model, for this we use import_export.fields.Field().